app/controllers/idv/in_person/address_controller.rb

```diff
@@ -109,11 +109,7 @@ def redirect_to_next_page
 
```

Comment:
I am adding this note here so it can be resolved. It is about your testing plan. It does not have anything to do with this file.

I think we should also test with hybrid remote flow; start on desktop, uploaded on mobile phone failing attention.yml files, and then going back to desktop to ensure we reach the new controller state ID page within IPP flow.

Reply:
Good call out! I've added a hybrid remote flow testing step to the plan 👍